tensorflow_hub/tools/make_image_classifier/make_image_classifier.py

```diff
@@ -153,6 +159,18 @@ def _assert_accuracy(train_result, assert_accuracy_at_least):
   else:
     raise AssertionError("ACCURACY FAILED:", accuracy_message)
 
+def _set_gpu_memory_growth():
+  gpus = tf.config.experimental.list_physical_devices('GPU')
+  if gpus:
+    try:
+      # Currently, memory growth needs to be the same across GPUs
+      for gpu in gpus:
+        tf.config.experimental.set_memory_growth(gpu, True)
+      logical_gpus = tf.config.experimental.list_logical_devices('GPU')
+      print(len(gpus), "Physical GPUs,", len(logical_gpus), "Logical GPUs")
+    except RuntimeError as e:
```

Comment:
The example code has the try-except block to illustrate what could go wrong with calling set_memory_growth. But why would it be needed here? I suggest to just drop it, unless there is something smart that can be done when the exception is caught.

Reply:
This is still unresolved. What happens if you just drop the try-statement and its except-block?